src/components/Navbar.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react';
-import phLogo from '../assets/logo.png'
+import phLogo from '../assets/logo_rgb.png'
 import '../styles/navbar.scss'
+import { SocialIcon } from 'react-social-icons';
```

Comment:
remove any unnecessary imports, especially SocialIcon. SocialIcon is not being used in this code.